### PR TITLE
Fixed typo in resiliency.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-WorkplaceAnalytics/issues/274.

### CODE REVIEW

File: Viva/insights/Use/resiliency.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The following are the percentage insights, their underlying metrics, and some de
...
 |Provide support |Percentage of employees who have less than 15 minutes of 1:1 time with their managers each week |[Meeting hours with manager 1:1](metric-definitions.md#meeting-hours-with-manager-1-1-define) |Percentage of employees who spend less than 15 minutes of weekly 1:1 time with their managers. To account for various meeting frequencies, the total time is calculated for each employee per month and averaged over a week. |
-|Connect across boundaries | Percentage of managers who spend a majority of their meeting time in double-booked meetings |[Conflicting meeting hours](metric-definitions.md#conflicting-meeting-hours-define) |Percentage of employees who more than 50 percent of their meeting hours in conflicting meetings. This insight is calculated weekly and averaged over all the weeks. |
+|Connect across boundaries | Percentage of employees who spend a majority of their meeting time in double-booked meetings |[Conflicting meeting hours](metric-definitions.md#conflicting-meeting-hours-define) |Percentage of employees who more than 50 percent of their meeting hours in conflicting meetings. This insight is calculated weekly and averaged over all the weeks. |
```
Comment (JohanFreelancer9) [inline Viva/insights/Use/resiliency.md:37]:
```suggestion
|Connect across boundaries | Percentage of employees who spend a majority of their meeting time in double-booked meetings |[Conflicting meeting hours](metric-definitions.md#conflicting-meeting-hours-define) |Percentage of employees who spend more than 50 percent of their meeting hours in conflicting meetings. This insight is calculated weekly and averaged over all the weeks. |
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**